rdwatch/scoring/views/animation.py

```diff
@@ -0,0 +1,210 @@
+import logging
+from collections.abc import Iterable
+
+from celery.result import AsyncResult
+from ninja import Router
+from pydantic import UUID4, BaseModel
+
+from django.core.files.storage import default_storage
+from django.db import transaction
+from django.http import HttpRequest
+from django.shortcuts import get_object_or_404
+
+from rdwatch.scoring.models import (
+    AnimationModelRunExport,
+    AnimationSiteExport,
+    EvaluationRun,
+)
+from rdwatch.scoring.tasks.animation_export import (
+    GenerateAnimationSchema,
+    create_modelrun_animation_export,
+    create_site_animation_export,
+)
+
+logger = logging.getLogger(__name__)
+
+router = Router()
+
+
+def generate_download_data(
+    exports: Iterable[AnimationSiteExport] | Iterable[AnimationModelRunExport],
+):
+    output = []
+    for export in exports:
+        if export.export_file:  # File exists it can be downloaded
+            output.append(
+                {
+                    'created': export.created,
+                    'completed': True,
+                    'arguments': export.arguments,
+                    'taskId': export.celery_id,
+                }
+            )
+        else:
+            task_id = export.celery_id
+            task = AsyncResult(task_id)
+            celery_data = {}
+            celery_data['state'] = task.state
+            celery_data['status'] = task.status
+            celery_data['info'] = (
+                str(task.info) if isinstance(task.info, Exception) else task.info
+            )
+            output.append(
+                {
+                    'created': export.created,
+                    'completed': False,
+                    'arguments': export.arguments,
+                    'taskId': export.celery_id,
+                    'state': celery_data,
+                }
+            )
+    return output
+
+
+@router.post('/site/{id}/')
+def generate_site_animation(
+    request: HttpRequest,
+    id: UUID4,
+    params: GenerateAnimationSchema,  # noqa: B008
+):
+    # Fetch the SiteEvaluation instance
+    task_id = create_site_animation_export.delay(
+        site_evaluation_id=id, settings=params.dict(), userId=request.user.pk
+    )
+    return task_id.id
+
+
+@router.get('/site/{id}/downloads/')
+def get_site_downloads(
+    request: HttpRequest,
+    id: UUID4,
+):
+    exports = AnimationSiteExport.objects.filter(
+        site_evaluation=id, user=request.user
+    ).order_by('-created')
+    return generate_download_data(exports)
+
+
+@router.get('/modelrun/{id}/downloads/')
+def get_modelrun_downloads(
+    request: HttpRequest,
+    id: UUID4,
+):
+    exports = AnimationModelRunExport.objects.filter(
+        configuration=id, user=request.user
+    ).order_by('-created')
+    logger.warning(f'Found: {exports.exists()} with id: {id}')
+    return generate_download_data(exports)
+
+
+class DeleteErrorSchema(BaseModel):
+    error: str
+
+
+class DeleteSuccessSchema(BaseModel):
+    success: str
```

Comment:
Instead of redefining these, can we also import them from the `core` app like the other schemas? They're basic classes, but I think it's still worth being consistent and it also enforces consistency between the two APIs.

```python
from rdwatch.core.views.animation import DeleteErrorSchema, DeleteSuccessSchema
```